main-site/src/pages/index.tsx

```diff
@@ -8,15 +8,18 @@ import { min } from '../../../shared-ui/lib/responsive';
 import { StyledDayBackground } from '../../../shared-ui/styled-components/background.style';
 import DayBackgroundDesktop from '../../../shared-ui/images/day-background-desktop.svg';
 import DayBackgroundMobile from '../../../shared-ui/images/day-background-mobile.svg';
+import Header from '../../../shared-ui/components/header/Header';
+import { homeTabInfo } from '../../../shared-ui/lib/data';
 
 const IndexPage: React.FC = () => {
   const isDesktop = useMatchMedia(min.tablet);
   return (
     <div>
+      <Header tabs={homeTabInfo} />
       {isDesktop ? (
-        <StyledDayBackground src={DayBackgroundDesktop} />
+        <StyledDayBackground src={DayBackgroundDesktop} alt='desktop day background'/>
```

Comment:
(might be bad practice oops, up to you whether to change) but unsure if alt text is needed here. It's a background so I don't think a textual description is necessary when images can't be displayed. Also where is the alt text displayed on the page when the background can't be rendered?